TestClient/MainWindow.cs

```diff
@@ -396,5 +401,21 @@ private void btnAdvanced_Click(object sender, EventArgs e)
             advanced.SetOBS(_obs);
             advanced.Show();
         }
+
+        private void browseButton_Click(object sender, EventArgs e)
+        {
+            DialogResult result = this.folderBrowserDialog1.ShowDialog();
+            if (result == DialogResult.OK)
+            {
+                string selectedPath = this.folderBrowserDialog1.SelectedPath;
+                folderpath.Text = selectedPath;
+                _obs.SetRecordingFolder(selectedPath);
```

Comment:
Redundant. Should only be in the "Set Path" button

Reply:
I thought that if they choose a folder with the picker that it's good to not need to set the path with the button after they've selected it through the folder picker

Reply:
That is confusing. And forcing a change a user may not want.

Reply:
understood, updated now to just update the textbox